src/estimates.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Calculate a weighted estimate for the interval until the next purchase
+ * Current purchase a tiny bit less weight than all previous purchases
+ * @param {Number} lastEstimate The last stored purchase interval estimate
+ * @param {Number} latestInterval The interval between the most recent and previous purchases
+ * @param {Number} numOfPurchases Total number of purchases for the item
+ */
+const calculateEstimate = (lastEstimate, latestInterval, numOfPurchases) => {
+  if (isNaN(lastEstimate)) {
+    lastEstimate = 14;
+  }
+
+  // FIXME algorithm doesn't work when there's only 1 purchase in the database
```

Comment:
Is FIXME a 'to do' type comment or a note of how the calculation will function?

Reply:
This was part of the script provided by the issue itself, so I'm not 100%. It appears it's a comment to revisit the algorithm in the future though, but it currently still runs fine without any additional change. Just maybe not optimally?